pkg/cluster/internal/kubeadm/config.go

```diff
@@ -117,7 +117,7 @@ func (c *ConfigData) Derive() {
 	for k := range c.RuntimeConfig {
 		runtimeConfigKeys = append(runtimeConfigKeys, k)
 	}
-	sort.Strings(featureGateKeys)
+	sort.Strings(runtimeConfigKeys)
```

Comment:
bad copy paste. the sorting is nice to have but not really strictly required currently.